src/main/java/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration.java

```diff
@@ -150,14 +172,48 @@ public FormValidation doCheckJwt(@QueryParameter String jwt) {
     }
 
     public List<Instance> getJiraInstances() {
-        return jiraInstances;
+        //Here, no authentication is done on each instance as it has been done during configuration
+        return this.jiraInstances.stream()
+                .map(formInstance -> {
+                    if(formInstance.getValue().equals(CLOUD_TYPE)){
+                        return getCloudInstance(formInstance);
+                    }
+                    return getServerInstance(formInstance);
+                })
+                .collect(Collectors.toList());
     }
 
-    public List<Instance> getJiraServerInstances() {
-        return jiraInstances.stream().filter(instance -> !instance.cloud()).collect(Collectors.toList());
+    public List<Instance> getJiraServerInstances()  {
+        try {
+            return crateJiraInstances(jiraInstances).stream().filter(instance -> !instance.cloud()).collect(Collectors.toList());
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return null;
```

Comment:
If we throw exceptions on the method `crateJiraInstances`, do we want to catch the exceptions and return "null"?